plugins/servicenow/help.md

```diff
@@ -1681,6 +1685,7 @@ Example output:
 
 # Version History
 
+* 8.0.0 - `Incident Created`: Resolved issue related to trigger not working. Updated SDK
```

Comment:
I think the text is duplicated from 7.4.1

Reply:
Ah man, thanks for spotting this as i've just noticed it hasn't pulled in some of my changes!